### PR TITLE
Alternative branching strategy for TestGymBuildkite

### PR DESCRIPTION
**Pull request by mironec**
#### Description
TestGym "release" branches will now just have names like "0.12.0".

#### Release note
N/A - internal

#### Tests
Will do

#### Documentation
N/A

#### Primary reviewers
@m-samiec

### CODE REVIEW

File: ci/ReleaseTool/ReleaseCommand.cs [hunk 1]
```diff
@@ -83,11 +86,45 @@ public ReleaseCommand(Options options)
...
         {
             Common.VerifySemanticVersioningFormat(options.Version);
-            var (repoName, pullRequestId) = Common.ExtractPullRequestInfo(options.PullRequestUrl);
+            var gitRepoName = options.GitRepoName;
             var gitHubClient = new GitHubClient(options);
-            var repoUrl = string.Format(Common.RepoUrlTemplate, options.GithubOrgName, repoName);
+            var repoUrl = string.Format(Common.RepoUrlTemplate, options.GithubOrgName, gitRepoName);
             var gitHubRepo = gitHubClient.GetRepositoryFromUrl(repoUrl);
 
+            if (options.PullRequestUrl == "" || options.PullRequestUrl == "\"\"")
```
Comment (joshuahuburn) [inline ci/ReleaseTool/ReleaseCommand.cs:94]: options.PullRequestUrl.Remove("\"").Empty()

File: ci/ReleaseTool/ReleaseCommand.cs [hunk 2]
```diff
@@ -83,11 +86,45 @@ public ReleaseCommand(Options options)
...
+                    {
+                        Logger.Error("The PullRequestUrl was empty or missing, but the release branch already exists, so presuming this step already ran.");
+                    }
+                    else
+                    {
+                        gitClient.CheckoutRemoteBranch(options.CandidateBranch);
+                        gitClient.ForcePush(options.ReleaseBranch);
+                    }
+
+                    gitClient.Fetch();
```
Comment (joshuahuburn) [inline ci/ReleaseTool/ReleaseCommand.cs:112]: This isn't strictly required

**Review by joshuahuburn — APPROVED** [review]
Couple suggestions

**Review by m-samiec — APPROVED** [review]

File: ci/ReleaseTool/ReleaseCommand.cs (hunk 1 above)
Comment (mironec) [reply ci/ReleaseTool/ReleaseCommand.cs]: Fixed

File: ci/ReleaseTool/ReleaseCommand.cs (hunk 2 above)
Comment (mironec) [reply ci/ReleaseTool/ReleaseCommand.cs:112]: Leaving it in for good luck

**Pull request merged**